holidayextras/ui-brand-heha
# HEHA-226 Update modal styling

#### What does this PR do? (please provide any background)
This PR cleans up modals within the heha brand. It takes us closer to the new design.

#### What tests does this PR have?
n/a

#### How can this be tested?
Install this branch of ui-brand-heha into tripapp/pattern-library and launch a modal on heha. It should look similar to this:

https://cloudup.com/clp2wiGy603

#### Screenshots / Screencast


#### What gif best describes how you feel about this work?
![](http://facweb.cs.depaul.edu/sgrais/images/Paint/kit3-Animated-300.gif#33980dd0-bded-11e5-8a00-cb4685777017)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
